tests/certification/go.mod

```diff
@@ -19,32 +19,33 @@ require (
 	github.com/cenkalti/backoff/v4 v4.2.1
 	github.com/cloudwego/kitex v0.5.0
 	github.com/cloudwego/kitex-examples v0.1.1
-	github.com/dapr/components-contrib v1.13.0-rc.6
-	github.com/dapr/dapr v1.13.0-rc.7
-	github.com/dapr/go-sdk v1.6.1-0.20231102031149-87bbb8cd690a
-	github.com/dapr/kit v0.13.0
+	github.com/dapr/components-contrib v1.13.1
+	github.com/dapr/dapr v1.13.4
+	github.com/dapr/go-sdk v1.6.0
```

Comment:
This may be the reason for the change that "does not seem right"... you've downgraded the go-sdk package